.github/workflows/agw-build-publish-container.yml

```diff
@@ -188,3 +188,69 @@ jobs:
       registry: ${{ needs.build-containers.outputs.registry }}
       test_targets: extended_tests
     secrets: inherit
+
+  publish-container-tests:
+    runs-on: ubuntu-20.04
+    if: always() && github.event_name == 'push'
+    needs: [test-containers-precommit, test-containers-extended]
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # pin@v3.1.0
+
+      - name: Collect all test results
+        run: |
+          mkdir -p lte/gateway/test_results
+          if [ -n ${{ needs.test-containers-precommit.outputs.final_status }} ]
+            cp ${{ needs.test-containers-precommit.outputs.test_results }} lte/gateway/test_results/
+            echo ${{ needs.test-containers-precommit.outputs.final_status }} > test_status_precommit.txt
+          else
+            echo fail > test_status_precommit.txt
+          fi
+          if [ -n ${{ needs.test-containers-extended.outputs.final_status }} ]
+            cp ${{ needs.test-containers-extended.outputs.test_results }} test_results/
+            echo ${{ needs.test-containers-extended.outputs.final_status }} > test_status_extended.txt
+          else
+            echo fail > test_status_extended.txt
+          fi
+
+      - name: Determine end result for both tests
+        run: |
+          diff -q test_status_precommit.txt test_status_extended.txt > /dev/null
```

Comment:
What is `-q` and is there a long form for this option?

Reply:
Nit: Then I would be in favour of removing it if it isn't needed.

Reply:
It was already removed after your first review.